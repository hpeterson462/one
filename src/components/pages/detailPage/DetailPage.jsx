import React from 'react';
import CharacterDetail from '../../characters/characterDetail/CharacterDetail';

export default function DetailPage() {
  return (
    <div>
      <CharacterDetail />
    </div>
  );
};
