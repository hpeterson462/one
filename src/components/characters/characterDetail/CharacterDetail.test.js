import React from 'react';
import { render, cleanup } from '@testing-library/react';
import DetailPage from '../../app/App';
import { MemoryRouter } from 'react-router-dom';

describe('CharacterDetail component', () => {
  afterEach(() => cleanup());
  it('renders character details', () => {
    const { asFragment } = render(<MemoryRouter>
      <DetailPage
        characters={[{
          id: '5cd99d4bde30eff6ebccfbbe',
          name: 'Adanel'
        }]}
      />
    </MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});
