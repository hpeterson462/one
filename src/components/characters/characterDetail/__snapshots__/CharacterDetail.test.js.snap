// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharacterDetail component renders character details 1`] = `<DocumentFragment />`;
